app/Plugin/ShibbAuth/Controller/Component/Auth/ApacheShibbAuthenticate.php

```diff
@@ -207,13 +218,18 @@ public function checkOrganization($org, $user)
 			$orgUserId = 1; //By default created by the admin
 			if ($user) $orgUserId = $user['id'];
 			$orgId = $organisations->createOrgFromName($org, $orgUserId, 0); //Created with local set to 0 by default
+			CakeLog::write('info', "User organisation ${org} created with id ${orgId}.");
+		}
+		else{
```

Comment:
please change it to look like this: } else {
so it's in one line and one space is before&after the else keyword.